src/components/ArrowButton.js

```diff
@@ -5,7 +5,7 @@ import { ReactComponent as Arrow } from '../assets/arrow-left.svg'
 function ArrowButton({ children, hotspot, onClick }) {
     return (
         <ArrowButtonStyled hotspot={hotspot} onClick={onClick}>
-            <Arrow hotspot={hotspot} className="arrow" />
+            <ArrowStyled hotspot={hotspot && 'hotspot'} />
```

Comment:
Oh, that is interesting. But I am not totally sure, if I can grasp it. What do you need the string 'hotspot' exactly for?

Reply:
Otherwise, it puts a true into the component, but I need hotspot as a prop.